src/Traits/HasSharedLogic.php

```diff
@@ -50,6 +50,30 @@ public function keyboardMarkup(array $markup): self
         return $this;
     }
 
+    /**
+     * unsets parse mode of the message.
+     *
+     * @return static
+     */
+    public function normal()
+    {
+        unset($this->payload['parse_mode']);
+
+        return $this;
+    }
+
+    /**
+     * Sets parse mode of the message.
+     *
+     * @return static
+     */
+    public function parseMode(string $mode = null)
```

Comment:
This allows for any invalid value to be provided. It must not allow that. Validation for only valid parse modes should be added

Reply:
We could use enum or array validation.